onnxruntime/core/providers/cuda/math/binary_elementwise_ops_impl.h

```diff
@@ -25,7 +25,8 @@ namespace cuda {
   BINARY_OP_NAME_EXPR(Or, (a | b))                   \
   BINARY_OP_NAME_EXPR(Xor, (a ^ b))                  \
   BINARY_OP_NAME_EXPR(PRelu, (a > (T)0 ? a : a * b)) \
-  BINARY_OP_NAME_EXPR(Compare, (a > b) ? 1 : 0)
+  BINARY_OP_NAME_EXPR(Compare, (a > b) ? 1 : 0)      \
```

Comment:
>Compare [](start = 22, length = 7)

not for this change, but I feel this should be named Greater?

Reply:
I didn't use greater because for Greater, by its spec, the output should be bool type, but here output is T type rather than bool. I chose Compare to distinguish the difference. If you prefer name it as Greater. I'm fine with that. @KeDengMS @HectorSVC